docs/source/asr/speaker_diarization/datasets.rst

```diff
@@ -1,72 +1,134 @@
 Datasets
 ========
-Check out page :doc:`Speech Classification Datasets <../speech_classification/datasets>` and :doc:`Speaker Recogniton Datasets <../speaker_recognition/datasets>` 
-for preparing datasets for training and validating VAD and speaker embedding models respectively.
 
-For Speaker Diarization inference, ``diarizer`` expects following arguments :
+This page is about preparing input dataset for diarization inference. To train or fine-tune speaker diarization system, the speaker embedding extractor should be trained or fine-tuned. Check out page :doc:`Speech Classification Datasets <../speech_classification/datasets>` and :doc:`Speaker Recogniton Datasets <../speaker_recognition/datasets>` 
+for preparing datasets for training and validating VAD and speaker embedding models respectively. 
 
-  - paths2audio_files
-  - oracle_num_speakers
-  - path2groundtruth_rttm_files [Optional]
-  - oracle_vad_manifest (if vad/rttm files are known)[Optional but suggested for good performace. This avoids FPR and TPR]
 
-paths2audio_files:
-  - a file containing absolute paths to audio files or list of paths to audio files. 
-  For generating a file that contains paths to audio files, you can simply use ``find`` bash command as shown below:
+Preparation of input data
+-------------------------
+
+Diarization inference is based on Hydra configurations which are fulfilled by .yaml files. See `NeMo Speaker Diarization Configuration Files <../configs>`_ for setting up the input configuration file for speaker diarization. Input data should be provided in line delimited JSON format as below:
+	
 .. code-block:: bash
 
-  find $PWD/{data_dir} -iname '*.wav' > path_to_audiofiles.txt
+  {"audio_filepath": "/path/to/audio_file", "offset": 0, "duration": null, label: "infer", "text": "-", "num_speakers": null, "rttm_filepath": "/path/to/rttm/file", "uem_filepath": "/path/to/uem/file"}
 
-.. note::
-  We expect audio and the corresponding RTTM files to have the same base name and the name should be unique (uniq-id).
+We refer to this file as manifest file and the path of the manifest file should be provided in Hydra configuration as follows:
 
-oracle_num_speakers:
-  - if number of speakers is known input it or pass null if not known. Accepts int or path to file containing uniq-id with num of speakers of that session 
+.. code-block:: bash
+   
+	diarizer.manifest_filepath="path/to/manifest/input_manifest.json"
 
-  Sample line (uniq-id `iaaa` session has 2 unique speakers )::
-    
-    iaaa 2
+In the input manifest file, audio_filepath item is required.
```

Comment:
This line can be moved up while mentioning about JSON format. Also better to change sentence as "in each line `audio_filepath` item is mandatory while rest of the items are optional and can be passed for desired diarization setting"

Reply:
Reflected this comment